web/src/main/java/se/inera/intyg/webcert/web/web/controller/facade/dto/ResourceLinkDTO.java

```diff
@@ -103,7 +124,8 @@ public boolean equals(Object o) {
             && Objects.equals(type, that.type)
             && Objects.equals(name, that.name)
             && Objects.equals(description, that.description)
-            && Objects.equals(body, that.body);
+            && Objects.equals(body, that.body)
+            && Objects.equals(title, that.title);
     }
 
     @Override
```

Comment:
Nice catch on updating equals and toString. Could you also update hashCode?

Reply:
Ops, missed that one! :-)

Done!